package.json

```diff
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "@originjs/vite-plugin-federation": "^1.3.6",
+    "bootstrap": "^5.3.3",
```

Comment:
Please utilize Tailwind's feature and remove bootstrap

Reply:
done